tests/playground.html

```diff
@@ -1460,7 +1542,45 @@ <h1>Blockly Playground</h1>
         </statement>
       </block>
     </category>
-    <category name="Fields" expanded="true">
+    <category name="Mutators">
```

Comment:
I think these (Mutator and Style categories) are dups.

Reply:
I'm not sure what you mean, so I'll just try to cover everything.

I checked and the categories are only showing up once:
![Categories](https://user-images.githubusercontent.com/25440652/72031038-04ea9680-3240-11ea-8414-31e6ef5a3177.png)
so that's rad.

But if you're wondering if the categories are no longer necessary, I'm not sure. You'd have to ask @alschmiedt

Reply:
I'm not sure what you mean so I'll just try to cover everything.

I checked and the categories are only showing up once:
![Categories](https://user-images.githubusercontent.com/25440652/72031140-53983080-3240-11ea-8951-f3f884df9391.png)
so that's rad.

But if you're wondering if the categories are no longer necessary, I'm not sure. You'd have to ask @alschmiedt

Reply:
I see what's going on. They're just being moved around. Mutators and Style categories were after Fields.
There was probably an intention to keep these alphabetically sorted, but I messed that up by adding Align second.
We could fix that here?